package.json

```diff
@@ -13,12 +13,16 @@
     "gatsby-plugin-react-helmet": "^3.0.0",
     "gatsby-source-contentful": "^2.1.78",
     "gatsby-transformer-remark": "^2.1.6",
+    "gatsby-transformer-sharp": "^2.4.6",
```

Comment:
I couldn't run `npm ci` in this branch, it seems that the dependencies were included here but not installed properly. Did the dependencies change? Maybe we can just include the change in scripts in this PR.

Reply:
Ran npm install to update `package-lock.json`. Should work now.